src/Layer/Layer.svelte

```diff
@@ -0,0 +1,45 @@
+<script>
+  import { getContext, setContext } from "svelte";
+
+  /**
+   * Specify the layer level and override any existing levels based on hierarchy.
+   * @type {0 | 1 | 2 }
+   */
+  export let level = undefined;
+
+  /**
+   * Specify the HTML element to render. If none is specified, a `div` is rendered.
+   * @type {typeof import("svelte").SvelteComponent}
```

Comment:
@SimpleProgrammingAU (docs/types still need another re-build at some point)

Reply:
Glad you raise this.

If I submitted all the changes that git tracks, it would be all the type files (right now, 167 files according to VSC).

Should we do a commit to v11/styles with a rebuild of the docs/types? Will that fix the issue in branches from v11?

Reply:
I was just referring to the fact that `types/Layer/Layer.svelte.d.ts` still has the outdated property information (still contains `id` and different type for `as`).

Not sure what those other changes are/when to include them.